Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -0,0 +1,297 @@
+                    MRTKOculusConfig configFile = Resources.Load<MRTKOculusConfig>("MRTK-OculusConfig");
+                    instance = Instantiate(configFile);
+
+                    if (instance == null)
+                    {
+                        UnityEngine.Debug.LogError("Failure to detect MRTK-OculusConfig. Please create an instance using the asset context menu, and place it in any Resources folder.");
+                    }
+                }
+                return instance;
+            }
+        }
+
+        [Header("Config")]
+        [SerializeField]
+        [Tooltip("Using avatar hands requires a local avatar prefab. Failure to provide one will result in nothing being displayed. \n\n" +
+                 "Note: In order to render avatar hands, you will need to set an app id in OvrAvatarSettings. Any number will do, but it needs to be set.")]
+        private bool renderAvatarHandsInsteadOfControllers = true;
+
+        /// <summary>
+        /// Using avatar hands requires a local avatar prefab. Failure to provide one will result in nothing being displayed.
+        /// </summary>
+        public bool RenderAvatarHandsInsteadOfController => renderAvatarHandsInsteadOfControllers;
+
+
+#if OCULUSINTEGRATION_PRESENT
+        [Header("Prefab references")]
+        [SerializeField]
+        [Tooltip("Prefab reference for OVRCameraRig to load, if none are found in scene.")]
+        private OVRCameraRig ovrCameraRigPrefab = null;
+
+        /// <summary>
+        /// Prefab reference for OVRCameraRig to load, if none are found in scene.
+        /// </summary>
+        public OVRCameraRig OVRCameraRigPrefab => ovrCameraRigPrefab;
+#endif
+        [SerializeField]
+        [Tooltip("Use this if you want to manage the avatar hands prefab yourself.")]
+        private bool allowDevToManageAvatarPrefab = false;
+
+        /// <summary>
+        /// Use this if you want to manage the avatar hands prefab yourself.
+        /// </summary>
+        public bool AllowDevToManageAvatarPrefab => allowDevToManageAvatarPrefab;
+
+        [SerializeField]
+        [Tooltip("Prefab reference for LocalAvatar to load, if none are found in scene.")]
+        private GameObject localAvatarPrefab = null;
+
+        [Header("Pointer Configuration")]
+        [SerializeField]
+        [Tooltip("Controls which teleport mode is utilized by MRTK-Quest controllers." +
+                 "Note to use the official pointer, you must add a parabollic pointer to your pointer input profile that supports articulated hands.")]
+        private TeleportPointerMode teleportPointerMode = TeleportPointerMode.Custom;
+
+        /// <summary>
+        /// Controls which teleport mode is utilized by MRTK-Quest controllers.
+        /// Note to use the official pointer, you must add a parabollic pointer to your pointer input profile that supports articulated hands.
+        /// </summary>
+        public TeleportPointerMode ActiveTeleportPointerMode
```

Comment:
Nit: Is there a reason why the private/public names of these things are different? i.e. what's the significance of "active" (is it just based on whether or not the platform is supported, and is this distinction interesting in the naming of the parameter?)